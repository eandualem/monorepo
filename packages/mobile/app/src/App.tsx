// eslint-disable-next-line import/default
import React, { FC } from "react";
import { SafeAreaView, Text } from "react-native";

const App: FC = () => {
  return (
    <SafeAreaView>
      <Text>Hello, World!</Text>
    </SafeAreaView>
  );
};

export default App;
