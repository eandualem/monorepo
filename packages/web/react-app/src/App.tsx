import { calcHelper, header, message } from "@monorepo/message";
import Link from "./components/Link";

const App = (): JSX.Element => {
  return (
    <>
      <h1>{header}</h1>
      <h2>{message}</h2>
      <p>{calcHelper(5, 5, 5, 5, 5, 5)}</p>
      <Link page="http://www.facebook.com">Facebook</Link>
    </>
  );
};

export default App;
