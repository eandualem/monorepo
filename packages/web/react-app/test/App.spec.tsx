import renderer from "react-test-renderer";
import App from "../src/App";

test("Link changes the class when hovered", () => {
  const component = renderer.create(<App />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
