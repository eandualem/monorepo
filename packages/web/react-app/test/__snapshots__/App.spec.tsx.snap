// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link changes the class when hovered 1`] = `
Array [
  <h1>
    Welcome to this monorepo.
  </h1>,
  <h2>
    Change this message to see the effect.
  </h2>,
  <p>
    The final result is 3125
  </p>,
  <a
    className="normal"
    href="http://www.facebook.com"
    onMouseEnter={[Function]}
    onMouseLeave={[Function]}
  >
    Facebook
  </a>,
]
`;
